models/elasticsearch.go

```diff
@@ -73,6 +73,7 @@ type ESSourceDocument struct {
 	Published       bool                `json:"published,omitempty"`
 	Language        string              `json:"language,omitempty"`
 	Survey          string              `json:"survey,omitempty"`
+	CanonicalTopic  string              `json:"canonical_topic"`
```

Comment:
Can I suggest we order fields alphabetically (unless there is a better logical order) in structs so it is easier to find fields

Reply:
Done.